### PR TITLE
New wheel for hsl()

### PR DESCRIPTION
**Pull request by teoli2003**
TEST DO NOT MERGE

### CODE REVIEW

**Comment by estelle** [comment]
i know it's still in draft, but it looks good!
A suggestion: make the live example (conic gradient) bigger.
A query: can the magenta-blues read "purples" 
 If you need to do them all, the magenta-reds could read "pinks".
    the cyan-greens could be aquas. the yellow-greens could be 'slime'.... kidding.

**Review by janbrasna — COMMENTED** [review]
What's the consensus on quality of image assets generally? This PNG seems quite low-res for 2022, I'd reckon a downsized `@2x` or an SVG (if the gradients don't yield prohibitively large file) might be more appropriate when the time comes for updating it.

_(If there's no better source file, I might try to draw a replacement if there's demand…)_

File: files/en-us/web/css/color_value/hsl/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ browser-compat: css.types.color.hsl
 ---
 {{CSSRef}}
 
-The **`hsl()`** functional notation expresses an {{glossary("RGB", "sRGB")}} color according to its _hue_, _saturation_, and _lightness_ components. An optional _alpha_ component represents the color's transparency.
+The **`hsl()`** functional notation expresses a {{glossary("RGB", "sRGB")}} color according to its _hue_, _saturation_, and _lightness_ components. An optional _alpha_ component represents the color's transparency.
```
Comment (janbrasna) [inline files/en-us/web/css/color_value/hsl/index.md:16]: The term is not "RGB" (that's the link), but "sRGB", so the sentence reads "expresses an sRGB color" therefore my gut feeling is the original was correct and this PR introduces a typo; but take that with a pinch of salt… from non-english-speaker.

**Comment by teoli2003** [comment]
Yes, I was not happy with the quality hence the fact it is still a draft. Also, I wasn't happy with the grey background color for the text. It is still on my todo but not too high as we have one for the moment.

If you are able to draw a better image, feel free to do it and I will be very happy to include yours instea.

**Comment by janbrasna** [comment]
Yes, the overall legibility is low, I'd be addressing better typography, contrast, resolution etc. (that's what tripped me out in the first place) — I'll try something and get back with PNG and hopefully even SVG to replace it with something editable for future use…

**Comment by teoli2003** [comment]
Feel free to open a new PR and ping me in it to review it.

**Comment by janbrasna** [comment]
Well I have tried making an SVG of the color space, but ended up with a file 3000% larger (and not even sure the gradients were to my liking, SVG-markup-wise…), so I don't have any better asset to offer right now unfortunately.

**Comment by janbrasna** [comment]
FYI this was the direction I was going for: _(just a mockup w/o correct angles and such)_

Screenshot 2022-06-26 at 20 40 05

But the actual gradient composition in SVG was something I couldn't overcome in a reasonable way so until I find someone more knowledgeable I'd say I'm pretty much stuck:/

**Comment by teoli2003** [comment]
I'm closing this PR: I have no time to work on this. Better images are welcome and can be proposed in a new PR.

**Pull request closed without merging**